src/scripts/start_valheim.sh

```diff
@@ -122,6 +122,11 @@ if [ ! "${TYPE}" = "vanilla" ]; then
   done
 fi
 
+if [ -d /start-valheim-post-install.d/ ]; then
```

Comment:
Mind of we change up the dir name and add quotes? 

Quotes for standardization and dir name for simplification. 

```bash
if [ -d "/valheim-post-install.d/" ]; then
  log "Executing post-install scripts"
  find /valheim-post-install.d/ -type f -executable -exec {} \;
fi
```